QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+$QUEST_JOBSTEP_20150317_000772	Rumpelstiltskin moved nimbly to chase the flying arrow, and after he got on, the arrow impaled the tree. As the arrow continued to oscillate, he danced on the arrow and sang an unknown tune. Lydia Schaffen again gauged her arrows, then she called ,"All of the pinecones from the pine tree on my front left side!" Upon calling the target, Lydia Schaffen unveiled her godlike archery skills. 
+$QUEST_JOBSTEP_20150317_000773	Her arrows poured faster than heavy rain during a hot summer day, as if the white tailed arrows were fired at the same time in the same direction. 
+$QUEST_JOBSTEP_20150317_000774	Rumpelstiltskin couldn't take it as calmly this round. Faster than the human eye can distinguish, Rumpelstiltskin sat on each arrow one by one and transfer to another arrow as the barrage cuts through the air. In the end, when he reached the last arrow before it reached the last pinecone, he was concerned he might crash into the target. Rumpelstiltskin, in his tired state, worked up a triumphant expression and spoke to Lydia Schaffen.
+$QUEST_JOBSTEP_20150317_000775	"So anyways, I managed to sit on every single arrow before the last arrow reached the target, therefore I win." Lydia Schaffen replied ," Certainly, I can't deny your word that you sat on every arrow before the last one reached it target." 
+$QUEST_JOBSTEP_20150317_000776	Rumpelstiltskin gave a wicked grin after hearing Lydia Schaffen's words and said ,"Therefore, I win this game. Well then, let's talk about the agreed compensation…" Before Rumpelstiltskin could finish, Lydia Schaffen shook her head and spoke ,"Don't tell me that because you won 2 rounds, you won't play the final round? I'm fine not doing it, but clearly we did promise each other to play 3 rounds, right?"
+$QUEST_JOBSTEP_20150317_000777	"The game doesn't end before the 3rd round, so I think it's pointless to talk about 'compensation' without the game ending…"
+$QUEST_JOBSTEP_20150317_000778	Recognizing that he has already won two rounds and had no reason to refuse the 3rd round, and for the reason that the game is certainly not over yet, Rumpelstiltskin abided to Lydia Schaffen's words. And so the final round begins, Lydia Schaffen measures her last arrow, and called out as she pulled the arrow back. "The target is… the moon in the sky!" Rumpelstiltskin got on the arrow and into a horseback riding posture as soon as the arrow left the bowstring, and not before long, he realized what he heard. 
+$QUEST_JOBSTEP_20150317_000779	And as he rode the arrow towards the moon, he also realized the fact that the game cannot end. As time passes, the arrow's power subsides and begin to fall towards the ground, however, Lydia Schaffen shot another arrow and caused the arrow that Rumpelstiltskin rode on to stop still, floating in the air.
```

Comment:
As time passes, the arrow's power subsides and begin to fall towards the ground -> As time passed, the arrow's power subsided and it began to fall towards the ground